core/src/test/java/edu/ucr/cs/riple/core/ConfigurationTest.java

```diff
@@ -160,6 +175,61 @@ public void test_required_flags_for_downstream_dependency_analysis_cli() {
     assertEquals(downstreamInfo, config.downstreamInfo);
   }
 
+  @Test
+  public void test_config_files_have_different_uuid() {
```

Comment:
```suggestion
  public void testConfigFilesHaveDifferentUUID() {
```

Reply:
d8227e8ba51b09112b2956aac622e5f498637618

Thanks for the suggestion, I will rename all other test methods outside of scope of this PR in a separate PR.